Symfony/CS/Tests/Fixer/Contrib/AlignDoubleArrowFixerTest.php

```diff
@@ -30,6 +30,74 @@ public function provideFixCases()
     {
         return array(
             array(
+                '$array = array(
+                "closure" => function ($param1, $param2) {
+                    return;
+                }
+            );', ),
+            array(
+                '<?php
+        return new JsonResponse(array(
+            "result" => "OK",
+            "html"   => 1, array(
+                "foo"    => "bar",
+                "foofoo" => array(
+                    "a"  => 1,
+                    "b"  => 2
+                )
+            ),)
+        );',
+                '<?php
+        return new JsonResponse(array(
+            "result" => "OK",
+            "html" => 1, array(
+                "foo" => "bar",
+                "foofoo" => array(
+                    "a" => 1,
+                    "b"  =>  2
+                )
+            ),)
+        );',
+            ),
+            array(
+                '<?php
+        return new JsonResponse([
+            "result" => "OK",
+            "html"   => renderView("views/my_view.html.twig", array(
+                "foo"    => "bar",
+                "foofoo" => 43,
+            )),
+        ]);',
+                '<?php
+        return new JsonResponse([
+            "result" => "OK",
+            "html" =>    renderView("views/my_view.html.twig", array(
+                "foo" => "bar",
+                "foofoo" => 43,
+            )),
+        ]);',
+            ),
+            array(
+                '<?php
+        return new JsonResponse([
+            "result" => "OK",
+            "html"   => renderView("views/my_view.html.twig", [
+                "foo"    => "bar",
+                "foofoo" => 42,
+            ]),
+            "baz" => "OK",
```

Comment:
Note to myself:
```php
<?php
    $data = [
        "foo"   ALIGNABLE1 => "Bar",
        "main"  ALIGNABLE1 => [array("baz"  ALIGNABLE3 => "Test")],
        "bar"   ALIGNABLE1 => array(),
    ];
    $data = array(
        "foo"   ALIGNABLE2 => "Bar",
        "Main"  ALIGNABLE2 => array("baz"  ALIGNABLE3 => "Test"),
        "bar"   ALIGNABLE2 => array(),
    );
    $var = [];
    foreach ($foo as $i => $bar) {
        $var[] = /* Comment */ [$i  ALIGNABLE4 => $bar];
    }
```

this is wrong on 1.10 and on the branch of this PR, the repeat of ALIGNABLE3 in two diff. groups causes the alignment to be done wrongly.